stock_auto_move/models/stock_auto_move.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# © 2014-2015 NDP Systèmes (<http://www.ndp-systemes.fr>)
+
+from openerp import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    auto_move = fields.Boolean(
+        "Automatic move",
+        help="If this option is selected, the move will be automatically "
+        "processed as soon as the products are available.")
+
+    @api.multi
+    def action_assign(self):
+        super(StockMove, self).action_assign()
+        # picking_env = self.env['stock.picking']
+        # Transfer all pickings which have an auto move assigned
+        moves = self.filtered(lambda m: m.state == 'assigned' and m.auto_move)
+        picking_ids = {m.picking_id.id for m in moves}
+        todo_pickings = self.env['stock.picking'].browse(picking_ids)
```

Comment:
Wouldn't it be better to use `todo_pickings = moves.mapped('picking_id')` ?
The `mapped` method already takes care of duplicates by using a set.